### PR TITLE
Add tutorial to results page

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds a tutorial to the Results page. Click the blue question icon in the bottom left to open the tutorial.

### CODE REVIEW

File: src/sass/_results.scss [hunk 1]
```diff
@@ -991,3 +991,16 @@ $results-container-width: 100% - $filter-container-width;
...
 }
+
+.tutorial-button {
+  bottom: 5px;
+  color: $primary-blue;
+  font-size: 2em;
+  left: 10px;
+  position: fixed;
+
+  > span {
```
Comment (elizabeth-jimenez) [inline src/sass/_results.scss:1002]: So i may learn - is there a benefit to using `>`?

File: src/Components/ResultsPage/ResultsPage.jsx [hunk 2]
```diff
@@ -27,14 +33,29 @@ class Results extends Component {
...
+    const { status } = data;
+    if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
+      // Need to set our running state to false, so we can restart if we click start again.
+      this.setState({ run: false });
+    }
+  };
+
+  handleTutorialButtonClick = () => {
+    const { isLoading, results } = this.props;
+    if (!isLoading && get(results, 'results', []).length) {
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsPage/ResultsPage.jsx:54]: Should we instead just hide the button on these conditions?

Comment (elizabeth-jimenez) [reply src/Components/ResultsPage/ResultsPage.jsx:54]: Good catch btw - i would have completely forgot about 0 results edge-case

File: src/sass/_betaHeader.scss [hunk 3]
```diff
@@ -62,7 +62,7 @@
     margin: 0;
     position: absolute;
     bottom: -19px;
-    z-index: 1000;
+    z-index: 100;
```
[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply src/sass/_betaHeader.scss:65]: Did you still want to do that in this PR?

File: src/Components/ResultsPage/tutorialSteps.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    content: 'Click "Add to Favorites" to add it to your personal list of favorites',
+    hideCloseButton: true,
+  },
+  {
+    target: '.compare-check-box-container',
+    content: 'Click "Compare" to compare up to five positions.',
+    hideCloseButton: true,
+  },
+  {
+    target: '.results-card-title-link',
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsPage/tutorialSteps.js:36]: `.results-card-title-link > a` will spotlight just the link 🦜 
image

**Comment by elizabeth-jimenez** [comment]
This is really cool 😎 !

Just a few comments - but looks good to go!

File: src/sass/_results.scss (hunk 1 above)
Comment (mjoyce91) [reply src/sass/_results.scss:1002]: `>` means only the first child span

```css
div {
  > span { color: blue; }
}
```

```html
<div>
  <span>
    I will be blue
    <span>
      I will not be blue
    </span>
  </span>
</div>
```

File: src/sass/_betaHeader.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (mjoyce91) [reply src/sass/_betaHeader.scss:65]: yes!

File: src/Components/ResultsPage/ResultsPage.jsx (hunk 2 above)
Comment (mjoyce91) [reply src/Components/ResultsPage/ResultsPage.jsx:54]: To me that felt weird - showing it and hiding it as results loaded. Let's just always show it, and only trigger the click event if there are results to work with.

File: src/Components/ResultsPage/tutorialSteps.js (hunk 4 above)
Comment (mjoyce91) [reply src/Components/ResultsPage/tutorialSteps.js:36]: Nice! Let me try it out

File: src/sass/_results.scss (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/sass/_results.scss:1002]: 😱  I never knew that scss would apply it to all `span` descendants. Good to know!

File: src/Components/ResultsPage/ResultsPage.jsx (hunk 2 above)
Comment (elizabeth-jimenez) [reply src/Components/ResultsPage/ResultsPage.jsx:54]: true and 'not having results' should be rare

**Comment by mjoyce91** [comment]
@elizabeth-jimenez - updates made

**Review by elizabeth-jimenez — APPROVED** [review]
👍

**Pull request merged**